CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree Android SDK Release Notes
 
+## Unreleased
```

Comment:
Is it worth calling out for the PayPal flow what the new behavior will be as a bug fix?

Reply:
Just added changelog entries for PayPal and LocalPayment 👍